### ISSUE TITLE
Angular 7.0 update

### ISSUE THREAD

**Issue by HackPoint** [codetok/cdk-admin#34]
First off all it's a great project! I'm using it and want to say many thanks.
I would like you to take under consideration:
1. ngular 7,0 improvements and changing it to use schematics and libraries.
2. It would require architecture redesign but will make it much more extensible.
3. And it would great if you will implement native CDK Material feature of virtual scrolling and dragging.
4. Grid implementation - ngx-datatable or other advanced grids.

**Comment by harikrishna266** [comment]
yes, There is a tone of things i want to improve about this theme. 
When i build this theme angular was at version 4, this was done specifically for a project as per that project requirement and most things were not planned well, i was also new to angular.
Time is really the only factor that is limiting from making this better, however I will try my best. 
I appreciate your feedback.
Thanks

**Comment by webpatrickkim** [comment]
Hey..i have successfully ran it but some components are not working locally.ie tables

**Comment by HackPoint** [comment]
@webpatrickkim what have you ran?

**Comment by webpatrickkim** [comment]
I mean am trying to use the dashboard in my project and when i open the
dashboard some links are not functional

**Comment by yeshwanthmn** [comment]
Can I add bootstrap in these theme
